docs/how_to_config.md

```diff
@@ -57,22 +57,69 @@ bit more "Pythonic" and easier to read.)
 This is a template for `.pre-commit-config.yaml`:
 
 ```yaml
+# Option 1: with native tool
 - repo: https://github.com/jsh9/pydoclint
   rev: <latest_tag>
   hooks:
     - id: pydoclint
       args:
-        - [--style=google, --check-return-types=False]
+        - --style=google
+        - --check-return-types=False
```

Comment:
Are you sure that this would work?

The official documentation uses `[]`: https://pre-commit.com/#arguments-pattern-in-hooks

The author of pre-commit also uses the same style when answering someone's question: https://github.com/pre-commit/pre-commit-hooks/issues/262#issuecomment-363927129

Reply:
Yeah YAML syntax allows for lists across bullets or on one line: https://stackoverflow.com/a/33136212

Do you prefer `[]` syntax? I don't care, as long as it stays within reasonable char width

Reply:
Thanks!  Bullet list looks nicer than `[]` I think.